sqeleton/databases/presto.py

```diff
@@ -141,11 +144,6 @@ def set_timezone_to_utc(self) -> str:
     def current_timestamp(self) -> str:
         return "current_timestamp"
 
-    def type_repr(self, t) -> str:
-        if isinstance(t, TimestampTZ):
-            return f"timestamp with time zone"
-        return super().type_repr(t)
```

Comment:
The second `type_repr` overrides the other, leading to incorrectly handling floats.